src/main/java/com/facebook/openwifirrm/modules/operationelement/OperationElement.java

```diff
@@ -0,0 +1,11 @@
+package com.facebook.openwifirrm.modules.operationelement;
```

Comment:
this should not be in "modules"

Reply:
Any suggestions for the best place to put it?

Reply:
For now you can use "com.facebook.openwifirrm.ucentral.operationelement"